lib/api/policy.go

```diff
@@ -93,6 +93,12 @@ type PolicySpec struct {
 	// 	deployment != "dev"
 	// 	! has(label_name)
 	Selector string `json:"selector" validate:"selector"`
+
+	// Untracked indicates whether packets matched by the rules in this policy should go through
+	// the data plane's connection tracking, such as Linux conntrack.  If True, the rules in
+	// this policy are applied before any data plane connection tracking, and packets allowed by
+	// this policy are marked as not to be tracked.
+	Untracked bool `json:"untracked,omitempty" validate:"omitempty"`
```

Comment:
Could you be more definitive? (As I'm not yet familiar with this detail myself.)